translation/lint.rst

```diff
@@ -33,6 +33,17 @@ The linter results can be exported to JSON using the ``--format`` option:
     $ php bin/console lint:yaml translations/ --format=json
     $ php bin/console lint:xliff translations/ --format=json
 
+When running the YAML linter inside `GitHub Actions`_, the output is automatically
+adapted to the format required by GitHub, but you can force that format too:
```

Comment:
I would not say "required". It's a nicer output in GitHub, but not required (the normal stdout works fine).